The CI found a problem when building for the gemini target:
````
  *
  * USB dual-mode controller drivers
  *
  Cadence USB Support (USB_CDNS_SUPPORT) [N/m/y/?] n
  Faraday FOTG210 USB2 Dual Role controller (USB_FOTG210) [M/n/y/?] m
    Faraday FOTG210 USB Host Controller support (USB_FOTG210_HCD) [N/y/?] (NEW) make[6]: *** [scripts/kconfig/Makefile:77: syncconfig] Error 1
make[5]: *** [Makefile:704: syncconfig] Error 2
make[4]: *** [Makefile:805: include/config/auto.conf.cmd] Error 2
make[3]: *** [Makefile:31: /__w/openwrt/openwrt/openwrt/build_dir/target-arm-openwrt-linux-muslgnueabi_musl/linux-gemini_generic/linux-6.1.61/.modules] Error 2
make[2]: *** [Makefile:11: compile] Error 2
time: target/linux/compile#20.47#10.73#24.42
````

--- a/package/kernel/linux/modules/usb.mk
+++ b/package/kernel/linux/modules/usb.mk
@@ -750,8 +750,9 @@ $(eval $(call KernelPackage,usb-serial-mct))
 
 define KernelPackage/usb-serial-mos7720
   TITLE:=Support for Moschip MOS7720 devices
-  KCONFIG:=CONFIG_USB_SERIAL_MOS7720
+  KCONFIG:=CONFIG_USB_SERIAL_MOS7720 CONFIG_USB_SERIAL_MOS7715_PARPORT=y
   FILES:=$(LINUX_DIR)/drivers/usb/serial/mos7720.ko
+  DEPENDS:=+kmod-ppdev
   AUTOLOAD:=$(call AutoProbe,mos7720)
   $(call AddDepends/usb-serial)
 endef

